src/lib/hooks/useStoredProfile.js

```diff
@@ -0,0 +1,36 @@
+import { useEffect, useRef, useState } from 'react'
+import { isEqual } from 'lodash'
+
+import GDStore, { useCurriedSetters } from '../undux/GDStore'
+import userStorage from '../gundb/UserStorage'
+
+export default (withLocalState = false) => {
```

Comment:
why do we need this flag? see rest of comments

Reply:
local profile state var is used in the ProfileEdit component.
In avatar view/cropper it's not used